catalog/admin/admin-poller-service/pom.xml

```diff
@@ -57,6 +57,11 @@
             <groupId>ddf.action.core</groupId>
             <artifactId>action-core-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>ddf.lib</groupId>
+            <artifactId>spock-shaded</artifactId>
+            <version>${ddf.version}</version>
```

Comment:
@adimka may need to look into the spock-shaded version here. I don't believe we had it on DDF 2.9.x. So you may have to actually put in the version here for the dependency to resolve correctly. I believe this is why the Bamboo build failed.